### DISCUSSION TITLE
v0.9.4.5

### DISCUSSION THREAD

**Discussion by pocopico** [pocopico/tinycore-redpill#295] [Fixes - Enhancements]
What are currently broken and needs fixing 👯

**Comment by damienmillet** [comment]
Hi ! it's nice idea

**Comment by 123tommyd** [comment]
Hi
What is the difference between sata boot or usb boot?

**Comment by pocopico** [comment]
If your loader boot media is a SATA disk (Usually on VMware) then you select SATA, otherwise if your boot media is USB (Usually on physical systems) you select USB.

**Comment by rimot-support** [comment]
Hello, we cannot find rplaoder and the other files in latest release from 9.x.x.x to 10. where are those files? Or do we made some mistakes?

**Reply by pocopico** [reply]
I'll have a look. That's strange though as all images are tested

**Reply by rimot-support** [reply]
Thanks for reply @pocopico we are waiting and looking forward any news from you

**Reply by pocopico** [reply]
Well I tested and the images look fine, what program did you use to write the image to the disk ?

**Comment by rimot-support** [comment]
Hello pocopico,

My hypervisor platform is vmware and I used vmdk disk option on vsphere 7.0
u3 vms, Before machine power on i converted disk using vmkfstool because
original vmdk is not working on vsphere.

19 Şub 2024 Pzt 00:16 tarihinde pocopico ***@***.***> şunu
yazdı:

**Comment by rimot-support** [comment]
Hello, 
Any News about the issue and its fixes?
Thanks

**Comment by rimot-support** [comment]
Hello,

Also i tried on VM Workstation pro and i keep disk format. After restart i can not found any concerned files. You can see in SSs. There is also no GUI just cli screen and no starter filles like as rploader etc.

![image](https://github.com/pocopico/tinycore-redpill/assets/119117618/b0087ef8-afcb-495b-a13c-206b88087635)
![image](https://github.com/pocopico/tinycore-redpill/assets/119117618/430e3c09-5dad-4afe-99b1-2811d61e80d0)

**Comment by alfista-sk** [comment]
Hi,
I have seen that you have also support for Synology DVA1622 but I dont know how to install it. Please is possible to help me?
Thanks.

**Reply by alfista-sk** [reply]
I have found how to install build it:
./rploader.sh build dva1622=7.2.0.-64570
 but it end swith an error
![tempImagefPybqL](https://github.com/user-attachments/assets/c855606b-1e36-455a-90f6-9710b15a34d9)
it looks that you dont have a some parts for downloading and building the DSM.
